nfpm.go

```diff
@@ -48,6 +50,18 @@ func Parse(in io.Reader) (config Config, err error) {
 	config.Info.Release = os.ExpandEnv(config.Info.Release)
 	config.Info.Version = os.ExpandEnv(config.Info.Version)
 
+	// parse the version as a semver so we can properly split the parts and support proper ordering for both rpm and deb
+	var v *semver.Version
+	if v, err = semver.NewVersion(config.Info.Version); err == nil {
+		config.Info.Version = fmt.Sprintf("%d.%d.%d", v.Major(), v.Minor(), v.Patch())
+		if config.Info.Release == "" && valid.IsInt(v.Prerelease()) {
```

Comment:
I would prefer to not import an entire lib just for int validation, could add a new func:

```go
func isInt(s string) bool {
  _, err := strconv.Atoi(v)
  return err == nil
}
```

and then:

```suggestion
		if config.Info.Release == "" && isInt(v.Prerelease()) {
```

Reply:
ok